README.md

```diff
@@ -94,6 +94,7 @@ export default Carousel
 
 ## Documentation
 
+- [How to Create a Hook](HOW-CREATE-A-HOOK.md)
```

Comment:
I think this should be linked in https://github.com/devthiago/react-ui-hooks/blob/master/CONTRIBUTING.md as well as part of the `Make the change`. What do you think?

Reply:
i think this could appear in both, what do you think? @talyssonoc @elainemattos @devthiago

Reply:
That's what I mean by "as well" 😛 , I agree!

Reply:
i'm sorry, i was thinking that you were talking to get this out of the readme :(

Reply:
at CONTRIBUTING.md
[...] 
- Make the tests.
- Check if the tests are all passing `yarn test`.
***Detailed explanation can be found at [How to Create a Hook](HOW-CREATE-A-HOOK.md)***

What do you think?

Reply:
at CONTRIBUTING.md i think you can change to something like that =>

### Create new approaches or make changes

- Make the changes or create a new Hook (Detailed explanation can be found at [How to Create a Hook]())

What do you think?

Reply:
Seems better, I'll make the changes